code/modules/mob/living/simple_animal/hostile/terror_spiders/procs.dm

```diff
@@ -0,0 +1,447 @@
+
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/DoShowOrders(var/usecache = 0, var/thequeen = null, var/theempress = null)
+	if(ckey)
+		to_chat(src, "------------------------")
+		if(ai_type == TS_AI_AGGRESSIVE)
+			to_chat(src, "Your Orders: <span class='danger'>kill all humanoids on sight! </span>")
```

Comment:
Not having this would lead to it reading as: "Your Orders:kill all"... which is bad.
Assuming you mean "move the space after the span" and doing that.